data/pid_data.py

```diff
@@ -2498,6 +2566,214 @@
+       'get_response': {'items': [{'name': 'fan_error_status', 'type': 'uint8',
+								  'labels': [(0, 'No Error'),
+											 (1, 'Main Fan Error'),
+											 (2, 'UPE Fans Error'),
+											 (3, 'Pinch Fan Error'),
+											 (4, 'Stack Fan Error'),
+											 (5, 'Stack Fan Error')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'FAN_ERROR_STATUS',
+       'set_request': {'items': [{'name': 'regulate_fan_speed', 'type': 'uint8',
+								 'labels': [(0, 'No Error'),
+											(1, 'Main Fan Error'),
+											(2, 'UPE Fans Error'),
+											(3, 'Pinch Fan Error'),
+											(4, 'Stack Fan Error'),
+											(5, 'Stack Fan Error')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33280},
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'max_lamp_power', 'type': 'uint8',
+								  'labels': [(1, '900 Watt'),
+											 (2, '1200 Watt'),
+											 (3, '1400 Watt')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'MAX_LAMP_POWER',
+       'set_request': {'items': [{'name': 'max_lamp_power', 'type': 'uint8',
+								 'labels': [(1, '900 Watt'),
+											(2, '1200 Watt'),
+											(3, '1400 Watt')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33792},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'fade_on_comm_loss', 'type': 'uint8',
+								  'labels': [(1, 'Hold'),
+											 (2, '30 Sec'),
+											 (3, '60 Sec')],
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'LAMP_FADE_ON_COMM_LOSS',
+       'set_request': {'items': [{'name': 'fade_on_comm_loss', 'type': 'uint8',
+								 'labels': [(1, 'Hold'),
+											(2, '30 Sec'),
+											(3, '60 Sec')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 33793},
+
+	  {'get_request': {'items': []},
+	   'get_response': {'items': [{'name': 'pan_lock', 'type': 'bool'}]},
+       'get_sub_device_range': 0,
+       'name': 'LOCK_PAN',
+       'set_request': {'items': [{'name': 'pan_lock', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34304},
+
+	  {'get_request': {'items': []},
+	   'get_response': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
+       'get_sub_device_range': 0,
+       'name': 'LOCK_TILT',
+       'set_request': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34305},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'free_motion', 'type': 'uint8',
+								  'labels': [(1, 'Normal Operation'),
+											 (2, 'Pan Tilt Free Motion'),
```

Comment:
Did a new commit.

FYI, this is how it looks in my xcode:


Mvg,

Bart van Stiphout

Mobile NL: +31648560121
Skype: bartvstiphout

www.bartvanstiphout.com <http://www.bartvanstiphout.com/>


	


Now available in the App Store:
Fixture Tester for iPhone/iPad/iPod Touch!
Find out more! <http://itunes.apple.com/us/app/fixture-tester/id368100165?mt=8>
> On 10 May 2015, at 01:07, Peter Newman <notifications@github.com> wrote:
> 
> In data/pid_data.py <https://github.com/OpenLightingProject/rdm-app/pull/20#discussion_r29997058>:
> 
> > +       'value': 34304},
> > +
> > +	  {'get_request': {'items': []},
> > +	   'get_response': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
> > +       'get_sub_device_range': 0,
> > +       'name': 'LOCK_TILT',
> > +       'set_request': {'items': [{'name': 'tilt_lock', 'type': 'bool'}]},
> > +       'set_response': {'items': []},
> > +       'set_sub_device_range': 0,
> > +       'link': '',
> > +       'value': 34305},
> > +			 
> > +	  {'get_request': {'items': []},
> > +       'get_response': {'items': [{'name': 'free_motion', 'type': 'uint8',
> > +								  'labels': [(1, 'Normal Operation'),
> > +											 (2, 'Pan Tilt Free Motion'),
> Nano and do it manually. :) Which editor do you use?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/OpenLightingProject/rdm-app/pull/20/files#r29997058>.
>